<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AKHILESG-IN" name="TestNGDemo.NewTest" tests="4" failures="0" timestamp="15 Mar 2020 09:31:33 GMT" time="3.027" errors="3">
  <testcase name="testing3" time="0.003" classname="TestNGDemo.NewTest"/>
  <testcase name="desting3" time="1.005" classname="TestNGDemo.NewTest">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.desting3() didn&#039;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.desting3() didn't finish within the time-out 1000
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:60)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- desting3 -->
  <testcase name="desting3" time="1.002" classname="TestNGDemo.NewTest">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.desting3() didn&#039;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.desting3() didn't finish within the time-out 1000
	at java.lang.Thread.currentThread(Native Method)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:64)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- desting3 -->
  <testcase name="desting3" time="1.017" classname="TestNGDemo.NewTest">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.desting3() didn&#039;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.desting3() didn't finish within the time-out 1000
	at org.testng.internal.BaseTestMethod.incrementCurrentInvocationCount(BaseTestMethod.java:666)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:69)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- desting3 -->
</testsuite> <!-- TestNGDemo.NewTest -->
